src/app/components/PrevButton.tsx

```diff
@@ -29,7 +29,7 @@ function PrevButton(props) {
       <button
         onClick={selectPrev}
         disabled
-        className="button button--primary button--disabled button--next"
+        className="button button--primary button--disabled button--next button--flex"
```

Comment:
Same here, I think we could use secondary (black outline)

Reply:
Changed to secondary button. Also moved removed these to files.